website/docs/r/elastic_cloud_elasticsearch.html.markdown

```diff
@@ -41,6 +41,10 @@ The following arguments are supported:
 
 * `sku_name` - (Required) Specifies the name of the SKU for this Elasticsearch. Changing this forces a new Elasticsearch to be created.
 
+**NOTE:** The SKU depends on the Elasticsearch Plans available for your account and is a combination of PlanID_Term.
```

Comment:
Convention in the docs is to prefix this with a `->` so that the Note is presented more prominently within a blue box 
```suggestion
-> **NOTE:** The SKU depends on the Elasticsearch Plans available for your account and is a combination of PlanID_Term.
```

Reply:
@poshett the way the docs are displayed in an IDE is different to how it's rendered on the Terraform Registry which is the target for these docs, see the example below for the `azurerm_arc_kubernetes_cluster` resource

Docs page: https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/arc_kubernetes_cluster
Source code: https://github.com/hashicorp/terraform-provider-azurerm/blob/main/website/docs/r/arc_kubernetes_cluster.html.markdown

Please update the prefix for the Note to `->`